dev-docs/analytics/yandex.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: analytics
+title: Yandex Metrica
+description: Track your Prebid.js in Yandex Metrica - Top-5 worldwide web analytics tool.
+modulecode: generic
```

Comment:
```suggestion
modulecode: yandexAnalytics
```

As your adapter is configured via `yandexAnalytics`, it's not the generic one mentioned here: https://docs.prebid.org/dev-docs/modules/genericAnalyticsAdapter.html right?